tests/misc_env/exec.py

```diff
@@ -152,6 +154,34 @@ def get_systemctl_service_name(node, service_name, command):
     return f"{command} {service_full_name}"
 
 
+def get_os_auth_commands(node):
+    """
+    Return list of export commands for openstack authentication like OS_USERNAME, OS_PASSWORD, OS_AUTH_URL etc...
+
+    Args:
+        node:       System to be used to gather the information.
+
+    Return:
+        command after adding the service name.
+    """
+    out, err = node.exec_command(cmd="cat ~/.config/openstack/clouds.yaml")
+    clouds_yaml = yaml.full_load(out)
+    auth = clouds_yaml["clouds"]["standalone"]["auth"]
+    commands = [
+        f"OS_USERNAME={auth['username']}",
+        f"OS_PASSWORD={auth['password']}",
+        f"OS_PROJECT_NAME={auth['project_name']}",
+        f"OS_PROJECT_DOMAIN_NAME={auth['project_domain_name']}",
+        f"OS_USER_DOMAIN_NAME={auth['user_domain_name']}",
+        f"OS_AUTH_URL={auth['auth_url']}",
+        f"OS_IDENTITY_API_VERSION={clouds_yaml['clouds']['standalone']['identity_api_version']}",
+        "export OS_USERNAME OS_PASSWORD OS_PROJECT_NAME OS_USER_DOMAIN_NAME OS_PROJECT_DOMAIN_NAME"
+        + " OS_AUTH_URL OS_IDENTITY_API_VERSION",
+    ]
+    cmd = ";".join(commands)
+    return cmd
+
+
 def run(ceph_cluster, **kwargs: Any) -> int:
```

Comment:
The exec.py file is used by a lot of scripts , can you attach pass log of 1 or 2 more tests which use this py file ?

Reply:
sure @TejasC88 , will provide more logs

Reply:
@TejasC88 , reverted back exec.py changes. created ~/os_auth.py file for openstack authentication on osp node.